releases_test.go

```diff
@@ -135,6 +232,22 @@ func Test_Releases(t *testing.T) {
 				}
 			}
 
+			providerRequests := []releaseRequest{}
+			{
+				var providerRequestsFile requestsFile
+				providerRequestsData, err := ioutil.ReadFile(filepath.Join(tc.provider, "requests.yaml"))
```

Comment:
Could we put some of these into like a `key.go`. 
I know you didn't introduce this, but it feels like we are spreading magic strings like `requests.yaml` all over the place.

Reply:
That's a pet peeve of mine too. Moved a bunch of the filenames out

Reply:
I wrote that code in a hurry. Sorry, thanks for cleaning up.